sympy/geometry/point.py

```diff
@@ -269,6 +269,38 @@ def distance(self, p):
         p = Point(p)
         return sqrt(sum([(a - b)**2 for a, b in zip(self.args, p.args)]))
 
+    def taxicab_distance(self, p):
+        """The Taxicab Distance from self to point p.
+
+        http://mathworld.wolfram.com/TaxicabMetric.html
```

Comment:
I would put this as a reference but not force the SymPy user to go there to know what the function does. The docstring should probably read, "Return the sum of the horizontal and vertical distances to point ``p``."